docker/testnet-node/insgorund.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM centos:7.6.1810 AS insgorund
```

Comment:
what about alpine?

Reply:
our base OS is centos, in derevative images you could use scratch as base image if you need small image size for running/pulling in production